qcommon/src/org/qcmg/common/util/AccumulatorUtils.java

```diff
@@ -464,6 +465,33 @@ else if (isRef)
 				return -1;
 			return o.quality - quality;
 		}
+		
+		//you need to implement the equals method thus overriding the default compareTo
+		@Override 
+		public boolean equals(Object o) { 
```

Comment:
don't think we need equals and hashcode for a utility class with only static methods

Reply:
I agree, and removed